pages/user guides/discover_nmap.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Discover your Hosts with Nmap
+title: Host discovery with Nmap
```

Comment:
Host or Host**s**?

Reply:
I think it should be Host. same as what we have on the Host discovery page.